app/src/main/java/edu/calvin/cs262/teamc/MyMatchesActivity.java

```diff
@@ -149,28 +158,22 @@ public View getView(int position, View convertView, ViewGroup parent) {
 
             View rowView = inflater.inflate(R.layout.match_list_item, parent, false);
 
-            ImageView pictureIv = (ImageView) rowView.findViewById(R.id.dogImageIv);
             TextView dogNameTv = (TextView) rowView.findViewById(R.id.dogNameTv);
             TextView dogBreedTv = (TextView) rowView.findViewById(R.id.dogBreedTv);
-            Log.e("getView-POsition", String.valueOf(position));
-            Log.e("getView-Values",String.valueOf(values.get(position).getImgSrc()));
-
-            Log.e("Size", String.valueOf(values.size()));
-
 
             dogNameTv.setText(values.get(position).getName());
             dogBreedTv.setText(values.get(position).getBreed());
 
 
```

Comment:
You have two empty lines here, maybe delete one?

Reply:
Good catch. To have one empty line is very helpful, to have "two looks like carelessness."